infrastructure/terraform.tfvars.example

```diff
@@ -4,3 +4,7 @@ project =
 # Set the zone. Note that us-central1-a is the cheapest, and you don't really
 # need to be near the edge given the application
 zone = "us-central1-a"
+project = ""
```

Comment:
Project and zone are duplicated

Please add comments for the two new variables.

Reply:
Actually the configuration files do not use region, so I'll remove it.